### DISCUSSION TITLE
jupyterlab-gitplus?

### DISCUSSION THREAD

**Discussion by kaipak** [ReviewNB/support#128] [General]
Hi, is this product still under active development? Looks like there hasn't been commits in the last month and see only minor updates to the repo all year. Thanks!

**Comment by amit1rrr** [comment]
Hello @kaipak 
Jupyterlab-Gitplus extension is maintained but no new features are being actively developed in the extension at this time. Please note, the extension currently on supports JupyterLab 2.x & 3.x. Some migration work would be required to make it work with JupyterLab 4.x

Contributions to the repository are always welcome.
